temporal/t.rast.accdetect/t.rast.accdetect.py

```diff
@@ -94,7 +94,7 @@
 # % description: A numerical suffix separated by an underscore will be attached to create a unique identifier
 # % required: yes
 # % multiple: no
-# % gisprompt:
+# #% gisprompt:
```

Comment:
Also to be deleted like below?

Reply:
Oh, left over from testing, thought I removed them. Will do...